DEVELOPING.md

```diff
@@ -66,6 +51,29 @@ Install the following plugins in your preferred editor
 
 For more script commands see `package.json`. Test coverage reports are available under reports/coverage.
 
-## Testing
+## Useful plugins
```

Comment:
Use Book Case for titles to stay consistent with the rest of the file.

Reply:
Gonna change them, since it feels a bit weird to have 1 or 2 character words uppercased.